fastlane/lib/fastlane/actions/gradle.rb

```diff
@@ -58,8 +58,8 @@ def self.run(params)
         apk_search_path = File.join(project_dir, '*', 'build', 'outputs', 'apk', '*.apk')
 
         # Our apk is now built, but there might actually be multiple ones that were built if a flavor was not specified in a multi-flavor project (e.g. `assembleRelease`), however we're not interested in unaligned apk's...
-        new_apks = Dir[apk_search_path].reject { |path| path =~ /^.*-unaligned.apk$/i}
-        new_apks = new_apks.map { |path| File.expand_path(path)}
```

Comment:
ah! this feels a little weird that there is an opening space but not a trailing one in the closure.